__tests__/app.test.js

```diff
@@ -139,18 +177,63 @@ describe("GET /api/reviews/:review_id", () => {
   })
   test("responds with 404 upon being given a valid but non-existent id", () => {
     return request(app)
-    .get('/api/reviews/26')
-    .expect(404)
-    .then(({ body }) => {
-      expect(body.msg).toBe("error: review id not found");
-    })
-  })
+      .get("/api/reviews/26")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("error: review id not found");
+      });
+  });
   test("responds with 400 when given an invalid id", () => {
     return request(app)
-    .get('/api/reviews/a')
-    .expect(400)
-    .then(({ body }) => {
-      expect(body.msg).toBe("error: invalid input");
-    })
-  })
-})
\ No newline at end of file
+      .get("/api/reviews/a")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("error: invalid input");
+      });
+  });
+});
+
+describe.only("GET /api/reviews/:review_id/comments", () => {
+  test("returns status code 200", () => {
+    return request(app).get("/api/reviews/2/comments").expect(200);
+  });
+  test("responds with an object with a key of comments containing an array", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
```

Comment:
Its not really necessary to test that it's an array if you're using a forEach on the array later on, forEach only works on arrays so if line 211 passes you know you have an array. It's good that you considered it though!